test/tools/ossfuzz/protoToYul.cpp

```diff
@@ -1074,7 +1094,15 @@ void ProtoConverter::visit(StoreFunc const& _x)
 		m_output << "mstore8(";
 		break;
 	}
-	visit(_x.loc());
+	// Write to memory within bounds, storage is unbounded
```

Comment:
Why is storage unbounded?

Reply:
Storage is unbounded because it is implemented as a key-value store. Memory needs to be bounded because the stack-to-memory mover needs a guarantee from the programmer that the program access memory within pre-defined bounds.